src/Classes/Daemons/MyRadio_PlaylistsDaemon.php

```diff
@@ -307,7 +307,7 @@ private static function updateLastFMGeoPlaylist()
                 'https://ws.audioscrobbler.com/2.0/?method=geo.getTopTracks&api_key='
                 .Config::$lastfm_api_key
                 .'&country='.Config::$lastfm_geo
-                .'&limit=150&format=json'
+                .'&limit=151&format=json'
```

Comment:
This was actually in a previous commit by @Eiron , and no idea how it either never got merged or why it's listed again.